gems/pending/util/MiqSockUtil.rb

```diff
@@ -8,6 +8,9 @@ def self.getHostName
   # Return the cannonical name for host in DNS
   def self.getFullyQualifiedDomainName
     Socket.gethostbyname(Socket.gethostname).first
+  rescue
+    $log.error "The appliance hostname could not be resolved. Please make sure to set a valid fully qualified hostname."
```

Comment:
@gmcculloug @Fryguy so maybe logging the nice exception here? https://github.com/Ladas/manageiq/blob/log_nice_message_about_unset_hostname/gems/pending/util/MiqSshUtilV2.rb#L258

That is the place called when we do SSA of OpenStack host